### PR TITLE
chore: move to go modules

### LINKED ISSUE #1043: Move to go mod

### PR DESCRIPTION
**Pull request by andrewrynhard**
This adds go.mod and go.sum and replaces dep with go modules. The
version of go bumped to 1.13 since modules were experimental in 1.11.

Fixes #1043.

### CODE REVIEW

**Comment by andrewrynhard** [comment]
@rmenn I update the import paths.

File: RELEASING.md [hunk 1]
```diff
@@ -32,15 +32,16 @@ Eventually we might want to consider moving the checkpointer to its own repo. Th
...
 - https://github.com/kubernetes/kubernetes/issues/489
-- https://github.com/kubernetes-incubator/bootkube/issues/424
+- https://github.com/kubernetes-sigs/bootkube/issues/424
 
 ## Updating Kubernetes Version
 
 ### Updating Kubernetes vendor code
 
-Vendoring currently relies on the [dep](https://github.com/golang/dep) tool. 
+Vendoring currently relies on the [dep](https://github.com/golang/dep) tool.
```
Comment (rmenn) [inline RELEASING.md]: nit: requires update since this moves to go -mod

**Pull request merged**